client/post-editor/editor-gutenberg-opt-in-notice/style.scss

```diff
@@ -41,6 +41,13 @@
 	}
 }
 
+@include breakpoint( '<660px' ) {
+	// We place the notice behind the sidebar on mobile devices
+	.focus-sidebar .editor-gutenberg-opt-in-notice {
+		z-index: z-index( 'root', '.editor-sidebar' );
```

Comment:
In hindsight, I'm not super happy about this. 🤔 
What do you think if, instead, we completely `display: none` the notice when the sidebar is open?